Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -314,16 +314,22 @@ if(![string]::IsNullOrEmpty($userEnteredMetadataDbName)){
 	$metadataDbName = $userEnteredMetadataDbName
 }
 
+$metadataConnStr = "Server=$($sqlServerAddress);Database=$($metadataDbName);Trusted_Connection=True;MultipleActiveResultSets=True;"
+
+Invoke-Sql $metadataConnStr "SELECT TOP 1 RoleID FROM CatalystAdmin.RoleBASE" | Out-Null
+Write-Success "Metadata DB Connection string: $metadataConnStr verified"
+
 $userEnteredIdentityDbName = Read-Host "Press Enter to accept the default Identity DB Name '$($identityDbName)' or enter a new Identity DB Name"
 if(![string]::IsNullOrEmpty($userEnteredIdentityDbName)){
```

Comment:
I know this isn't part of these changes, but can't the user enter a DB name with spaces or other forbidden characters? If so, is that handled?